chainerrl/links/noisy_linear.py

```diff
@@ -26,10 +28,22 @@ def __init__(self, mu_link, sigma_scale=0.4):
 
         with self.init_scope():
             self.mu = mu_link
+            self.mu.W.initializer = VarianceScalingUniform(1)
```

Comment:
`chainer.initializers.LeCunUniform(1/np.sqrt(3))` can be used.

Reply:
I never thought of that!